DiscordRPC/DiscordRPC.csproj

```diff
@@ -6,7 +6,7 @@
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Is a Newtonsoft.Json update required? This is a major update too, and for compatibility reasons I try to avoid updating this library.

Reply:
yeah fixed that